lib/phoenix_live_reload/live_reloader.ex

```diff
@@ -142,11 +142,20 @@ defmodule Phoenix.LiveReloader do
   defp reload_assets_tag(conn, endpoint, config) do
     path = conn.private.phoenix_endpoint.path("/phoenix/live_reload/frame#{suffix(endpoint)}")
 
-    if class = config[:iframe_class] do
-      ~s[<iframe src="#{path}" class="#{class}"></iframe>]
+    iframe_attrs = config
+    |> Keyword.get(:iframe_attrs, [])
+    |> Keyword.put_new(:src, path)
+    |> Keyword.put_new(:hidden, true)
+
+    iframe_attrs = if Keyword.has_key?(config, :iframe_class) do
+      Keyword.put_new(iframe_attrs, :class, config[:iframe_class])
```

Comment:
Should the `iframe_class` be deprecated in favor of `iframe_attrs`? If yes this would need a deprecation warning.